react-charts/line-chart/step6.md

```diff
@@ -1,56 +1,48 @@
-Now we'll add axis labels to the line chart...
+Add a legend to the line chart.
 
-## Task
+1) <strong>Make sure the `App.js` file is still open.</strong>
 
-1) Make sure the App.js file is still open
+2) <strong>Locate the code for the `padding` property in the `<Chart>` component.</strong>
 
-2) Locate the code that looks like the following:
+It should look like this:
 
 <pre class="file">
 &lt;Chart
   constrainToVisibleArea
   containerComponent={&lt;ChartVoronoiContainer labels={({ datum }) =&gt; `${datum.name}: ${datum.y}`} /&gt;}
   maxDomain={{y: 10}}
   minDomain={{y: 0}}
+  padding={{
+    bottom: 75,
+    left: 75,
+    right: 50,
+    top: 50
+  }}
   height={250}
   width={600}
 &gt;
 </pre>
 
-3) Add the following property to that section:
+3) <strong>Update the padding for the right value to accommodate the legend.</strong>
 
-<pre class="file" data-target="clipboard">
-padding={{
-  bottom: 75,
-  left: 75,
-  right: 50,
-  top: 50
-}}
-</pre>
-
-- This will add more padding to the bottom and left properties to accommodate the axis labels
-
-4) Locate the code that looks like the following:
+`right: 200`
 
-<pre class="file">
-&lt;ChartGroup&gt;
-...
-&lt;/ChartGroup&gt;
-</pre>
+4) <strong>Add the `legendOrientation` and `legendPosition` properties to the `<Chart>` component.</strong>

```

Comment:
Split up the `legendOrientation` into its own step
the `legendPosition` into its own step
and the `legendData` into its own step